package.json

```diff
@@ -100,6 +100,9 @@
     "sourceror": "^0.8.5",
     "unified": "^11.0.0",
     "uuid": "^11.0.2",
+    "workbox-core": "^7.3.0",
```

Comment:
Is this required for anything in particular that we need to have it explicitly as a dependency?

Reply:
Yeah, needed in the service worker. Previously it was a transitive dependency under CRA so we didn't need to explicitly put it in package.json